packages/core/src/js/feedback/FeedbackWidget.tsx

```diff
@@ -311,6 +318,17 @@ export class FeedbackWidget extends React.Component<FeedbackWidgetProps, Feedbac
               </TouchableOpacity>
             </View>
           )}
+          {config.enableScreenshot && !this.state.attachmentUri && (
+            <TouchableOpacity style={styles.takeScreenshotButton} onPress={() => {
+              // eslint-disable-next-line @typescript-eslint/no-var-requires
+              const { hideFeedbackButton, showScreenshotButton } = require('./FeedbackWidgetManager');
```

Comment:
The cyclic dependencies are properly fixed with a small refactoring in a followup PR https://github.com/getsentry/sentry-react-native/pull/4718